MapboxNavigationTests/InstructionsBannerViewIntegrationTests.swift

```diff
@@ -69,6 +71,27 @@ class InstructionsBannerViewIntegrationTests: XCTestCase {
 
         super.tearDown()
     }
+    
+    func testCustomVisualInstructionDelegate() {
+        let view = instructionsView()
+        view.instructionDelegate = TextReversingDelegate()
```

Comment:
`instructionDelegate` will be deallocated immediately because it's weak. Keep a local ref to `TextReversingDelegate()`. (×2)